docker/sawtooth-pdo.sgx.yaml

```diff
@@ -47,5 +47,5 @@ services:
       - ${PDO_SGX_KEY_ROOT:-./sgx/}:/project/pdo/build/opt/pdo/etc/keys/sgx/
       - /var/run/aesmd:/var/run/aesmd
     devices:
-      - /dev/isgx:/dev/isgx
```

Comment:
just a reminder for this one -- rest looks good

Reply:
Not sure what you are talking about, no such diff exists ...



.... (now :-)

Reply:
aha!